frontend/packages/console-shared/src/utils/resource-utils.ts

```diff
@@ -160,21 +162,27 @@ const sortByRevision = (
 };
 
 const getAnnotation = (obj: K8sResourceKind, annotation: string): string => {
```

Comment:
nit: Not new, but these utilities should probably use `K8sResourceCommon` which has more strict types.

```suggestion
const getAnnotation = (obj: K8sResourceCommon, annotation: string): string => {
```